src/pydata_sphinx_theme/__init__.py

```diff
@@ -382,7 +382,9 @@ def _get_local_toctree_for(
         kwargs["maxdepth"] = int(kwargs["maxdepth"])
     kwargs["collapse"] = collapse
 
-    for toctreenode in doctree.traverse(addnodes.toctree):
+    # Can just use "findall" once docutils 0.18+ is required
```

Comment:
```suggestion
    # FIX: Can just use "findall" once docutils 0.18+ is required
```

Just so it is easier to find with `grep -r FIX`.